graph/telemetry/istio/appender/mesh_check.go

```diff
@@ -1,44 +1,46 @@
 package appender
 
 import (
+	"context"
 	"github.com/kiali/kiali/config"
 	"github.com/kiali/kiali/graph"
 )
 
-const SidecarsCheckAppenderName = "sidecarsCheck"
+const MeshCheckAppenderName = "meshCheck"
+const SidecarsCheckAppenderName = "sidecarCheck"
```

Comment:
```suggestion
const SidecarsCheckAppenderName = "sidecarsCheck"
```

Typo.  If we want back compat we need to keep the exact same name constant.